man/man8/zfsconcepts.8

```diff
@@ -162,37 +163,44 @@ If needed, ZFS file systems can also be managed with traditional tools
 If a file system's mount point is set to
 .Sy legacy ,
 ZFS makes no attempt to manage the file system, and the administrator is
-responsible for mounting and unmounting the file system. Because pools must
+responsible for mounting and unmounting the file system.
+Because pools must
 be imported before a legacy mount can succeed, administrators should ensure
 that legacy mounts are only attempted after the zpool import process
-finishes at boot time. For example, on machines using systemd, the mount
-option
+finishes at boot time.
+For example, on machines using systemd, the mount option
 .Pp
 .Nm x-systemd.requires=zfs-import.target
 .Pp
 will ensure that the zfs-import completes before systemd attempts mounting
-the filesystem. See systemd.mount(5) for details.
+the filesystem.
+See
+.Xr systemd.mount 5
+for details.
 .Ss Deduplication
 Deduplication is the process for removing redundant data at the block level,
-reducing the total amount of data stored. If a file system has the
+reducing the total amount of data stored.
+If a file system has the
 .Sy dedup
-property enabled, duplicate data blocks are removed synchronously. The result
+property enabled, duplicate data blocks are removed synchronously.
+The result
 is that only unique data is stored and common components are shared among files.
 .Pp
-Deduplicating data is a very resource-intensive operation. It is generally
-recommended that you have at least 1.25 GiB of RAM per 1 TiB of storage when
-you enable deduplication. Calculating the exact requirement depends heavily
+Deduplicating data is a very resource-intensive operation.
+It is generally recommended that you have at least 1.25 GiB of RAM
+per 1 TiB of storage when you enable deduplication.
+Calculating the exact requirement depends heavily
 on the type of data stored in the pool.
 .Pp
 Enabling deduplication on an improperly-designed system can result in
-performance issues (slow IO and administrative operations). It can potentially
-lead to problems importing a pool due to memory exhaustion. Deduplication
-can consume significant processing power (CPU) and memory as well as generate
-additional disk IO.
+performance issues (slow IO and administrative operations).
+It can potentially lead to problems importing a pool due to memory exhaustion.
+Deduplication can consume significant processing power (CPU) and memory as well
+as generate additional disk IO.
 .Pp
 Before creating a pool with deduplication enabled, ensure that you have planned
 your hardware requirements appropriately and implemented appropriate recovery
-practices, such as regular backups. As an alternative to deduplication
-consider using
-.Sy compression=on ,
-as a less resource-intensive alternative.
+practices, such as regular backups.
+As an alternative to deduplication consider using the
+.Sy compression
+property as a less resource-intensive alternative.
```

Comment:
Alternative is duplicated in this sentence (as in original). How about:

    Consider using the
    .Sy compression
    property as a less resource-intensive alternative.

Reply:
Yeah